plaidml/edsl/tests/edsl_test.cc

```diff
@@ -55,6 +55,20 @@ Tensor Softmax(const Tensor& X) {
   return E / N;
 }
 
+Tensor ConstAdd() {
+  std::vector<int64_t> shape = {4};
+  std::vector<int> a = {4, 3, 2, 1};
+  std::vector<int> b = {1, 2, 3, 4};
+  std::string current_device = plaidml::Settings::get("PLAIDML_DEVICE");
```

Comment:
Use const ref so we don't make a copy of the string. 
```
const auto& curDevice = 
```

nit: use `camelBack` instead of `snake_case`.